### DISCUSSION TITLE
404 page appears after deployment from Github both on Preview and the official site.

### DISCUSSION THREAD

**Discussion by sc0ttferren** [vercel/community#1662] [Help]
### Whenever I deploy a Github project onto Vercel, both the preview _and_ my site only has a 404 pop up, I don't know what to do to not let this happen, because it always happens to me and probably some others too.
![image](https://user-images.githubusercontent.com/100329425/222170920-ecc21780-e7d2-4df6-a6a2-54abb780d0e6.png)

### Additional information

**404: NOT_FOUND
Code: NOT_FOUND
ID: cle1::4xmqv-1677681268313-e312aeb6379e**

### Example

https://github.com/EliMendez19/SafeSearch

**Comment by amyegan** [comment]
Hi @EliMendez19! In a fun twist of irony, I'm getting a 404 when I try to see your example repo. Can you make the example public or re-share the corrected link?
